app.js

```diff
@@ -15,8 +17,25 @@ app.use("*", (req, res) => {
 
 //PSQL errors
 
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ message: "400 - Bad request" });
+  }
+  next(err);
+});
+
 //Custom errors
 
+app.use((err, req, res, next) => {
+  if (err.status && err.message) {
+    console.log(err.status);
```

Comment:
dont forget to remove console.logs once you have completed a task :)

Reply:
Yep! I've been through and removed them - but I think I've removed them on the next branch.